packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselAutoplay.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { makeStyles, tokens, typographyStyles } from '@fluentui/react-components';
+import {
+  Carousel,
+  CarouselButton,
+  CarouselCard,
+  CarouselNav,
+  CarouselNavImageButton,
+  CarouselSlider,
+  CarouselAutoplayButton,
+} from '@fluentui/react-carousel-preview';
+import * as React from 'react';
+
+const SWAP_IMAGE = 'https://fabricweb.azureedge.net/fabric-website/assets/images/wireframe/image-square.png';
+const useClasses = makeStyles({
+  test: {
+    ...typographyStyles.largeTitle,
+    alignContent: 'center',
+    borderRadius: tokens.borderRadiusLarge,
+    height: '450px',
+    textAlign: 'center',
+  },
+});
+
+const TestComponent: React.FC<{ accentColor: string; children: string }> = props => {
+  const { accentColor, children } = props;
+  const classes = useClasses();
+
+  return (
+    <div className={classes.test} style={{ backgroundColor: accentColor }}>
+      {children}
+    </div>
+  );
+};
+
+export const Autoplay = () => {
+  return (
+    <div>
+      <Carousel groupSize={1}>
+        <CarouselSlider>
+          <CarouselCard>
+            <TestComponent accentColor="#B99095">Card 1</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#FCB5AC">Card 2</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#B5E5CF">Card 3</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#3D5B59">Card 4</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#F9EAC2">Card 5</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#FEE7E6">Card 6</TestComponent>
+          </CarouselCard>
+          <CarouselCard>
+            <TestComponent accentColor="#FFD898">Card 7</TestComponent>
+          </CarouselCard>
+        </CarouselSlider>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'center',
+          }}
+        >
```

Comment:
When I took these for a spin in NVDA - the nav and auto play buttons all just read as "button".... maybe we need a separate issue to track this, but I feel like they should have a little more information for screen reader users.

Reply:
wrapping in tool tips may solve that.

Reply:
AFAIK we don't provide default aria labels in the component itself due to localization, but I'll make sure I add them to the demo code as part of the upcoming accessibility PR